doc-assets/node/18005-search-videos/18005.html

```diff
@@ -589,79 +484,188 @@ <h3>Combining terms</h3>
+        <td><code>q=tags:bee,bop</code></td>
+        <td>returns videos with tag "bee" OR "bop"</td>
+      </tr>
+      <tr>
+        <td><code>q=tags:bee tags:bop</code></td>
+        <td><code>q=tags:bee%20tags:bop</code></td>
+        <td>returns videos with tag "bee" OR "bop"</td>
+      </tr>
+      <tr>
+        <td><code>q=+tags:bee tags:bop</code></td>
+        <td><code>q=%2Btags:bee%20tags:bop</code></td>
+        <td>all videos returned must have tag "bee"; they may have tag "bop" as well</td>
+      </tr>
+      <tr>
+        <td><code>q=+tags:bee +tags:bop</code></td>
+        <td><code>q=%2Btags:bee%20%2Btags:bop</code></td>
+        <td>all videos returned will have tag "bee" AND tag "bop"</td>
+      </tr>
+      </tbody>
+    </table>
+  </section>
+
+  <!-- Search for specfic videos -->
+  <section class="bcls-section">
+    <h2 id="specificVideos">Search specific videos</h2>
+
+    <p>If you want to limit your search to a specific set of videos, you can do so searching on <code>id</code>:</p>
+
+    <p>Example: This request returns videos with ids <code>123456789</code>, <code>987654321</code> and <code>48376387</code>
+
+    <pre class="line-numbers"><code class="language-http">q=id:123456789%20id:987654321%20id:48376387</code></pre>
+  </section>
+
+  <!-- State Search -->
+  <section class="bcls-section">
+    <h2 id="searchByState">Search by state</h2>
+
+    <p>You can perform or filter searches by the state of the video using the parameter:</p>
+
+    <pre class="line-numbers"><code class="language-http">q=state:ACTIVE( | INACTIVE | PENDING | DELETED)<sup><strong><a href="#note3">[3]</a></strong></sup></code></pre>
+
+    <h3>Notes</h3>
+
+    <ul>
+      <li id="note3"><sup><strong>[3]</strong></sup> Search for DELETED videos is only available for videos deleted in the past 10 days (the current time minus 10 days), and only through the CMS API (not the Playback API).</li>
+    </ul>
+  </section>
+
+  <!-- Stemming -->
+  <section class="bcls-section">
+    <h2 id="Stemming">Stemming</h2>
+
+    <p>Stemming is supported, but <strong>not</strong> partial word searches. For example, <code>q=name:ban</code> should return videos with the names "<code>Parking Ban Announced</code>" or "<code>Parking to be Banned</code>" or "<code>City Banning Parking</code>" but not "<code>Bank Holiday</code>" or "<code>Bandit Captured</code>".</p>
+  </section>
+
+  <!-- Special Characters/Encoding -->
+  <section class="bcls-section">
+    <h2 id="url_encoding">Spaces/Special Characters</h2>
+
+    <p>The CMS API generally handles special characters in search strings, with a couple of exceptions:</p>
+
+    <ul>
+      <li>Spaces are not allowed, and must be encoded as <code>%20</code>.
+
+        (<code>+</code> signs can also replace spaces, but this can lead to confusion in your queries as <code>+</code> can also indicate that a term is required.  See  <a href="#requiredexcludedterms">required/excluded syntax</a>)
```

Comment:
might be worth laying out that the confusion is due to unescaped `+`s or just include a link to the relevant section in the doc. This may read like our system will get confused.

Reply:
Not really sure how to word it much better without confusing the issue further.  There is a link to the `required` section.